### PR TITLE
[286905] Migrate from Cacoo diagrams to drawio.png

### PR DESCRIPTION
**Pull request by aDaikiKamata**
https://gate.tok.access-company.com/redmine/issues/286905#note-14

### CODE REVIEW

File: doc_src/assets/AntikytheraArchitecture.drawio.png
Comment (Double-oxygeN) [inline doc_src/assets/AntikytheraArchitecture.drawio.png:1]: These symlinks are broken. Please run `mix docs` and check that the images are displayed correctly.

**Review by Double-oxygeN — CHANGES_REQUESTED** [review]

File: doc_src/assets/AntikytheraArchitecture.drawio.png
Comment (aDaikiKamata) [reply doc_src/assets/AntikytheraArchitecture.drawio.png:1]: Sorry, I didn't notice the warnings in the `mix docs`.
Fixed at https://github.com/access-company/antikythera/pull/271/commits/96fd2723800596149a03bb635032cf5dee7ea775 and checked these images display correctly.

I even forgot to delete the `AntikytheraDeploymentFlow.png`.
Fixed at https://github.com/access-company/antikythera/pull/271/commits/e3af5a1e7c118a613e511289170ffd8bc277f7ab.

**Comment by aDaikiKamata** [comment]
@Double-oxygeN Thank you for your review.
I made some revisions. Would you be able to check them?

**Review by Double-oxygeN — APPROVED** [review]
LGTM

**Review by aYukiSekiguchi — APPROVED** [review]
LGTM

**Pull request merged**